xclim/sdba/measures.py

```diff
@@ -296,7 +296,7 @@ def _rmse_internal(_sim: xr.DataArray, _ref: xr.DataArray) -> xr.DataArray:
         input_core_dims=[["time"], ["time"]],
         dask="parallelized",
     )
-    out = out.assign_attrs(units=ensure_delta(ref.units))
+    out = out.assign_attrs(units=ref.units, units_metadata={"temperature: difference"})
```

Comment:
all measures and properties are independent of units.  It needs to be removed here and in `properties._annual_statistic` also.

Reply:
Indeed, fixed.